### PR TITLE
Flexible aws

### PR DESCRIPTION
**Pull request by chris-salemove**
Expose bunch of settings.

### CODE REVIEW

**Comment by indrekj** [comment]
+1

**Comment by madis** [comment]
:heart:

File: cluster/aws/util.sh [hunk 1]
```diff
@@ -717,7 +715,9 @@ function kube-up {
     EXISTING_CIDR=$($AWS_CMD describe-subnets --subnet-ids ${SUBNET_ID} --query Subnets[].CidrBlock --output text)
     echo "Using existing CIDR $EXISTING_CIDR"
     INTERNAL_IP_BASE=${EXISTING_CIDR%.*}
-    MASTER_INTERNAL_IP=${INTERNAL_IP_BASE}${MASTER_IP_SUFFIX}
+    if [[ -z "$MASTER_INTERNAL_IP" ]]; then
```
Comment (deiwin) [inline cluster/aws/util.sh]: trailing whitespace :P

**Comment by madis** [comment]
@chris-salemove I suggest using some plugin to automatically remove trailing whitespace. Most of us are using it, I'm IntelliJ has something too :)

**Pull request merged**